GOVERNANCE.md

```diff
@@ -90,22 +89,62 @@ in terms of project direction, to potentially less experienced contributors.
-* Make decisions when regular community discussion doesn’t produce consensus on an issue in a reasonable time frame.
+* Make decisions when regular community discussion does not produce consensus on an issue
+  in a reasonable time frame.
 * Make decisions about strategic collaborations with other organizations or individuals.
 * Make decisions about the overall scope, vision and direction of the project.
 * Developing funding sources
 * Deciding how to disburse funds with consultation from Core Contributors
 
 The council may choose to delegate these responsibilities to sub-committees. If so, Council members must update this document to make the delegation clear.
 
-Note that individual council member does not have the power to unilaterally wield these responsibilities, but the council as a whole must jointly make these decisions. In other words, Council Members are first and foremost Core Contributors, but only when needed they can collectively make decisions for the health of the project.
-
-ArviZ will be holding its first election to determine its initial council in the coming weeks and this document will be updated.
+Note that individual council member does not have the power to unilaterally wield these
+responsibilities, but the council as a whole must jointly make these decisions.
+In other words, Council Members are first and foremost Core Contributors, but only
+when needed they can collectively make decisions for the health of the project.
+
+### Council Decision Making Process
+By and large we expect the decisions in ArviZ to be made _ad hoc_ and require little formal
+coordination and with the community at large. However for controversial proposals and
+new core contributors the council may need to intervene to make the final decision
+in a group vote.
+
+#### Call for a vote
+Core contributors can call for a vote to resolve a target issue they feel has
+been stale for too long and informal consensus appears unlikely. For a voting
+to be called, the target issue must be at least 2 months old.
+
+To do so, they have to open a proposal issue ticket labeled "Council Vote".
+The proposal issue should contain a link to the target issue and a proposal
+on how to resolve it. Proposals should include a statement making clear what it
+means to "agree" or to "disagree".
+
+Before voting starts, at least 3 days will be left for Core
+Contributors to raise doubts about the proposal's _phrasing_, no extra discussion will take
+place in the proposal issue. Proposal issues should be locked from creation
+to prevent attracting discussion from people not familiar with the decision
+process.
+
+#### Voting process
+
+* Each council member will vote either Yes, No, or Neutral.
+* It is recommended that all council members expose their reasons when voting.
+  "No" votes however _must_ list the reasons for disagreement. Any "no" vote
+  with no reason listed will be considered a "neutral" vote.
```

Comment:
How about now @AlexAndorra ? It is true that the reasons behind a yes vote may not be the same as the reasons listed in the proposal, but I still think that requiring no votes to list their reasons is not making it more difficult to vote no. I'll try to explain better why and provide some example.

Proposals are submitted by core contributors who are trusted members of the community to the point they have merge privileges and have had to go through a selection process mediated by the council. We should therefore expect them to have ArviZ best interests in mind. 

Therefore, I feel that voting "yes" without listing any reasons (or even voting yes because the proposal aligns with your idea) does not do much harm to the project nor the community. Voting no on the other hand goes along way towards knocking down the proposal. Knocking down proposals can make the people who proposed not good enough, that they do not belong, not worthy of being a core contributor... _especially if the proposal is knocked down without explaining why_